samples/TodosApi.Dapper/TodosApi.cs

```diff
@@ -52,7 +52,9 @@ await db.QuerySingleOrDefaultAsync<Todo>("SELECT * FROM Todos WHERE Id = @id", n
         if (!inputTodo.IsValid)
             return Results.Extensions.ValidationProblem(inputTodo.Errors);
 
-        return await db.ExecuteAsync($"UPDATE Todos SET Title = @Title, IsComplete = @IsComplete WHERE Id = {id}", inputTodo.Value) == 1
+        inputTodo.Value.Id = id;
```

Comment:
This generates a nullability warning

Using the C# 11 we could add the `??` in the `Validated<Todo> inputTodo` argument

Reply:
Not yet, would rather wait until it's out of preview before introducing that. Let's just go with `inputTodo.Value?.Id = id;` or introducing a nullability check for now.